### PR TITLE
Bump webhook to v0.2.7-rc2

### PR DESCRIPTION
**Pull request by KevinJoiner**

### CODE REVIEW

**Review by MbolotSuse — DISMISSED** [review]
LGTM but you should get a review from Team 3 as well.

File: packages/rancher-webhook/package.yaml [hunk 1]
```diff
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/webhook/releases/download/v0.2.6/rancher-webhook-0.2.6.tgz
-version: 1.0.5
+url: https://github.com/rancher/webhook/releases/download/v0.2.7-rc2/rancher-webhook-0.2.7-rc2.tgz
```
Comment (prachidamle) [inline packages/rancher-webhook/package.yaml:1]: Is there a corresponding rancher-webhook ticket that describes what is getting updated or fixed in this version?

Comment (prachidamle) [reply packages/rancher-webhook/package.yaml:1]: this was discussed offline.

**Comment by prachidamle** [comment]
LGTM, PR needs to be rebased to resolve conflicts.

**Review by prachidamle — DISMISSED** [review]

**Review by MbolotSuse — APPROVED** [review]
LGTM

**Review by prachidamle — APPROVED** [review]
LGTM

**Pull request merged**